src/test/java/emissary/kff/KffDataObjectHandlerTest.java

```diff
@@ -51,4 +84,49 @@ public void testMapWithPrefix() {
         Map<String, String> m = kff.hashData(DATA, "name", "foo");
         assertTrue("Prefix prepends on normal key names but we got " + m.keySet(), m.get("foo" + KffDataObjectHandler.KFF_PARAM_MD5) != null);
     }
+
+    @Test
+    public void testHashMethod() {
+        kff = new KffDataObjectHandler(true, true, true);
+        payload.setParameter(KFF_PARAM_KNOWN_FILTER_NAME, "test.filter");
+        kff.hash(payload);
+        assertEquals("test.filter", payload.getStringParameter(KFF_PARAM_BASE + "FILTERED_BY"));
+        assertTrue(KffDataObjectHandler.hashPresent(payload));
+        assertEquals(DATA_MD5, payload.getStringParameter(KFF_PARAM_MD5));
+        assertEquals(DATA_CRC32, payload.getStringParameter(KFF_PARAM_BASE + "CRC32"));
+        assertEquals(DATA_SSDEEP, payload.getStringParameter(KFF_PARAM_SSDEEP));
+        assertEquals(DATA_SHA1, payload.getStringParameter(KFF_PARAM_SHA1));
+        assertEquals(DATA_SHA256, payload.getStringParameter(KFF_PARAM_SHA256));
+        assertEquals(KFF_DUPE_CURRENT_FORM, payload.getFileType());
+        assertArrayEquals(payload.data(), new byte[0]);
+    }
+
+    @Test
+    public void testParentToChildMethod() {
+        // String value = "value";
+        payload.setParameter(KFF_PARAM_DUPE_HIT, payload);
+        KffDataObjectHandler.parentToChild(payload);
+        assertNull(payload.getStringParameter(KFF_PARAM_DUPE_HIT));
+        // assertEquals(thing, parent.getParameter(KFF_PARAM_PARENT_HIT));
```

Comment:
I'm assuming this is meant to be removed or reworked in an update.

Reply:
I can't recall what I was using this for or what I was going to do with this... I will remove it for now.